payments/tests/test_payments_mutations.py

```diff
@@ -0,0 +1,262 @@
+import random
+import uuid
+from decimal import Decimal
+
+import pytest
+
+from berth_reservations.tests.utils import (
+    assert_doesnt_exist,
+    assert_field_missing,
+    assert_not_enough_permissions,
+)
+from payments.enums import PriceUnits
+from payments.models import BerthProduct
+from payments.schema import BerthPriceGroupNode, BerthProductNode
+from resources.schema import HarborNode
+from utils.relay import to_global_id
+
+CREATE_BERTH_PRODUCT_MUTATION = """
+mutation CREATE_BERTH_PRODUCT($input: CreateBerthProductMutationInput!) {
+    createBerthProduct(input: $input) {
+        berthProduct {
+            id
+            priceValue
+            priceUnit
+            priceGroup {
+                name
+            }
+            harbor {
+                id
+            }
+        }
+    }
+}
+"""
+
+
+@pytest.mark.parametrize(
+    "api_client", ["berth_services"], indirect=True,
+)
+def test_create_berth_product(api_client, berth_price_group, harbor):
+    variables = {
+        "priceValue": str(
+            Decimal(round(random.uniform(1, 999), 2)).quantize(Decimal("1.00"))
```

Comment:
since you are casting this to string anyway for the sake of simplicity I would just do 
```python
"priceValue": str(round(random.uniform(1, 999), 2)),
```
here and in all similar places